### PR TITLE
Sd 5318 - Solving the issue with Cursor jumping to the beginning of pasted text in the caption of an image

### PR DESCRIPTION
**Pull request by lnogues**
Solving the issue with Cursor jumping to the beginning of pasted text in the caption of an image

### CODE REVIEW

File: scripts/superdesk/editor2/editor.js [hunk 1]
```diff
@@ -566,11 +566,23 @@ function SdTextEditorBlockEmbedController($timeout, editor, renditions, config)
             vm.model.caption = caption;
             // update the caption in the view
             vm.caption = caption;
+
             // on change callback
             $timeout(function() {
                 vm.onBlockChange();
             });
         },
+        handlePasta: function(e) {
```
Comment (petrjasek) [inline scripts/superdesk/editor2/editor.js]: I guess it should be `handlePaste`

**Comment by petrjasek** [comment]
:+1: looks good

File: scripts/superdesk/editor2/editor.js (hunk 1 above)
Comment (pavlovicnemanja) [inline scripts/superdesk/editor2/editor.js]: Mmmm finally, handling food with Superdesk :) I prefer meat, but pasta also works for me :)

**Comment by pavlovicnemanja** [comment]
👍

**Comment by mdhaman** [comment]
@petrjasek @lnogues change looks good. I hope this change does not break our formatters. I am just being cautious. Would it be possible to move this change to separate branch and test it?

**Pull request merged**